lib/msal-node/src/client/ManagedIdentityClient.ts

```diff
@@ -73,6 +75,43 @@ export class ManagedIdentityClient {
         );
     }
 
+    private allEnvironmentVariablesAreDefined(
+        environmentVariables: Array<string | undefined>
+    ): boolean {
+        return Object.values(environmentVariables).every(
+            (environmentVariable) => {
+                return environmentVariable !== undefined;
+            }
+        );
+    }
+
+    /**
+     * Determine the Managed Identity Source based on available environment variables. This API is consumed by ManagedIdentityApplication's getManagedIdentitySource.
+     * @returns AzureIdentitySdkManagedIdentitySourceNames - Azure Identity SDK defined identifiers for the Managed Identity Sources
```

Comment:
```suggestion
     * @returns AzureIdentitySdkManagedIdentitySourceNames - Identifying Azure source.
```

Reply:
Or something. These are not defined by Azure Identity

Reply:
Good catch, I forgot to update that description